app/views/content/how-to-apply-for-teacher-training/teacher-training-personal-statement.md

```diff
@@ -40,73 +40,95 @@ quote:
 
-Your personal statement should include:
+Teacher training personal statements can be up to 1000 words. 90% of successful candidates write 500 words or more.  
 
-* why you want to be a teacher
+It can take time to write a good personal statement, but you do not have to write it in one go. A teacher training adviser can help you structure your personal statement so you can write it in manageable sections.  
 
-* why you want to teach the subject or specialism you're applying for - if you're applying for primary, describe why you want to teach this age group
+Your personal statement is a professional application, so you’ll need to check your spelling and grammar.   
 
-* the experience and personal characteristics you have, and why they would make you a great teacher 
+### What to include in your personal statement  
 
-You can include any information you think is relevant to demonstrate your knowledge, qualifications, and experience. Teacher training providers want to see your passion and that you understand what a career in teaching involves.
+You can include any information you think is relevant to demonstrate your knowledge and experience.   
 
-### How to write your personal statement
+As well as showing your passion for teaching, your statement should include:  
 
-When writing your personal statement you should make sure you check your spelling and grammar in your application. You want to make the best possible impression.
+* evidence of research and understanding of the role of a teacher and the national curriculum  
 
-You can use ChatGPT or other artificial intelligence (AI) tools to help you write your personal statement. You should not rely on it to write your entire statement because:
+* relevant experience, either in a classroom or working with young people in another setting  
 
-* AI tools use bland language and will not be able to give details about you as a person. Using them may result in your application being unsuccessful
+* transferable skills and how you can bring these skills into the classroom  
 
-* your account to apply for teacher training may be blocked if you consistently submit personal statements that look like they have been written with AI tools
+* why you want to teach your chosen subject and examples of how you’ll do this  
 
+### Using AI to help write a personal statement   
 
-## Do I use the same personal statement for each application?
+You can use ChatGPT or other artificial intelligence (AI) tools to help you write your personal statement.   
 
-You can use the same personal statement for every course you apply to.
+You should not rely on it to write your entire statement though. AI tools use bland language and produce generic results which will not make your statement personal to you.   
 
-However, there may be some instances where you'd like to tailor it to different courses.
+Using AI tools may result in your application being unsuccessful.   
```

Comment:
I know this is similar to what is in prod already but I think it is slightly odd - first we say you can use AI tools, then we say using them may result in you being unsuccessful

Worth discussing with Apply team whether we can be clearer here?